blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Our IntoxiCODED cloudy summer
+layout: post
+created_at: Thu September 21 2017
```

Comment:
Your blog post is scheduled on Tuesday September 26 2017. Can you please change a date everywhere?